packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/setBilling.ts

```diff
@@ -33,7 +33,7 @@ const makeContractInput = async (input: CommandInput): Promise<ContractInput> =>
     config: {
       observation_payment_gjuels: new BN(input.observationPaymentGjuels).toNumber(),
       transmission_payment_gjuels: new BN(input.transmissionPaymentGjuels).toNumber(),
-      recommended_gas_price_uluna: input.recommendedGasPriceUluna
+      recommended_gas_price_uluna: new BN(input.recommendedGasPriceUluna).toString(),
```

Comment:
If this works, I think it will have the effect of converting the decimal an integer, either by truncating or rounding a suppose.  Is that really what we want to do?

From:  https://github.com/indutny/bn.js/
"Note: decimals are not supported in this library."

Reply:
You're completely right. I'll open a fix

Reply:
Looks like it's handled in https://github.com/smartcontractkit/chainlink-terra/pull/76